### DISCUSSION TITLE
Is there any way to share/reuse field conditions?

### DISCUSSION THREAD

**Discussion by taviks** [directus/directus#16771] [Archive]
I have a very generic field condition that I'd like to reuse on all of my fields. 

Is there any way to only create this once and then reuse it? Or, do I have to recreate it for every field?

**Comment by paescuj** [comment]
As far as I know, there is currently no possibility to reuse condition sets. So this could be an idea / feature request 👍

What you could do, however, is to use the [API](https://docs.directus.io/reference/introduction.html) to create the conditions where you would be able to reuse the definitions.
